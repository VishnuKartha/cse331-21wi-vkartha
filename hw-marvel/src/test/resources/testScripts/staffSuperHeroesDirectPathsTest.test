# Tests that the several direct path connection in the staff SuperHeroes Test file works

# To run this test, the staffSuperHeroes dataset must exist at
#   hw-marvel/src/main/resources/data/staffSuperheroes.tsv

LoadGraph staffSuperheroes staffSuperheroes.tsv

FindPath staffSuperheroes Ernst-the-Bicycling-Wizard Notkin-of-the-Superhuman-Beard
FindPath staffSuperheroes Ernst-the-Bicycling-Wizard Perkins-the-Magical-Singing-Instructor
FindPath staffSuperheroes Ernst-the-Bicycling-Wizard Grossman-the-Youngest-of-them-all
FindPath staffSuperheroes Perkins-the-Magical-Singing-Instructor Notkin-of-the-Superhuman-Beard
FindPath staffSuperheroes Perkins-the-Magical-Singing-Instructor Grossman-the-Youngest-of-them-all
FindPath staffSuperheroes Notkin-of-the-Superhuman-Beard Grossman-the-Youngest-of-them-all





