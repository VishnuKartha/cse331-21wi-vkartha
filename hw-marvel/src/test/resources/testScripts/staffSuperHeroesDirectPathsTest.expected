# Tests that the several direct path connection in the staff SuperHeroes Test file works

# To run this test, the staffSuperHeroes dataset must exist at
# hw-marvel/src/main/resources/data/staffSuperheroes.tsv

loaded graph staffSuperheroes

path from Ernst-the-Bicycling-Wizard to Notkin-of-the-Superhuman-Beard:
Ernst-the-Bicycling-Wizard to Notkin-of-the-Superhuman-Beard via CSE331
path from Ernst-the-Bicycling-Wizard to Perkins-the-Magical-Singing-Instructor:
Ernst-the-Bicycling-Wizard to Perkins-the-Magical-Singing-Instructor via CSE331
path from Ernst-the-Bicycling-Wizard to Grossman-the-Youngest-of-them-all:
Ernst-the-Bicycling-Wizard to Grossman-the-Youngest-of-them-all via CSE331
path from Perkins-the-Magical-Singing-Instructor to Notkin-of-the-Superhuman-Beard:
Perkins-the-Magical-Singing-Instructor to Notkin-of-the-Superhuman-Beard via CSE331
path from Perkins-the-Magical-Singing-Instructor to Grossman-the-Youngest-of-them-all:
Perkins-the-Magical-Singing-Instructor to Grossman-the-Youngest-of-them-all via CSE331
path from Notkin-of-the-Superhuman-Beard to Grossman-the-Youngest-of-them-all:
Notkin-of-the-Superhuman-Beard to Grossman-the-Youngest-of-them-all via CSE331